src/http/client.rs

```diff
@@ -4475,9 +4475,9 @@ impl Http {
     /// let channel_id = ChannelId::new(381880193700069377);
     /// let route = Route::ChannelMessages { channel_id };
     ///
-    /// let mut request = Request::new(route, LightMethod::Post).body(Some(bytes));
+    /// let mut request = Request::<'_, 0>::new(route, LightMethod::Post).body(Some(bytes));
```

Comment:
I don't like doing this. The only solution I can think of would be to make `params` a required argument to `new` - then the value of `N` could be inferred. Thoughts?